techmatters/hrm
# CHI-1912: Trigger sqs resource import

## Description

* When resources are imported, we publish a message to reindex them onto the queue
* Currently the logic is NOT to reject the whole transaction if publishing to the reindex queue fails, but to log an error instead - which 

### Checklist

### Related Issues
CHI-1912

### Verification steps

* Deploy this code to dev
* Delete the current import progress record from the Development DB for the Aselo Development
* When the next import run occurs, it should reimport everything, and submit all those records for indexing
